packages/@okta/vuepress-site/docs/reference/api/brands/index.md

```diff
@@ -1095,6 +1095,989 @@ curl -v -X DELETE \
 HTTP/1.1 204 No Content
 ```
 
+## Email template operations
+
+The Email Templates API allows you to programatically manage email customizations.
```

Comment:
So we're okay with calling this a separate API?

Reply:
I just followed the prevailing convention in the doc - e.g., Themes API.